src/components/App/App.js

```diff
@@ -18,7 +20,17 @@ class App extends Component {
 	}
 
 	confirmMatch(index, img) {
-		if (!this.state.prediction) this.setState({prediction: {index, img}});
+		this.setState(
+			prevState =>
+				prevState.prediction ? null : {prediction: {index, img}}
+		);
+	}
+
+	componentDidMount() {
+		loadUsers(users => {
```

Comment:
As discussed offline we could pass this in as a prop instead of importing it, this lets us test and mock this much easier